test/worker.test.js

```
@@ -0,0 +1,151 @@
+    });
+  });
+
+  describe('GET /info', () => {
+    it('should return client and edge information', async () => {
+      const { req, ctx } = createRequest('GET', '/info');
+      req.cf = {
+        city: 'Test City',
+        country: 'US',
+        colo: 'DFW'
+      };
+      const res = await worker.fetch(req, env, ctx);
```

ESLint

[error] 'env' is not defined.

(no-undef)